src/testUtils.js

```diff
@@ -0,0 +1,74 @@
+/**
+ *  Copyright (c) 2016-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* global expect, define, it */
+
+import fs from 'fs';
+import path from 'path';
+
+import jscodeshift from './core';
```

Comment:
this should be a lazy require inside of `runTest`. Jest resets the module registry after each test and we had issues previously where two copies of ast-types was present.

cc @pieterv